radware-alteon/info.json

```diff
@@ -0,0 +1,246 @@
+                "editable": true
+            },
+            {
+                "title": "Password",
+                "type": "password",
+                "name": "password",
+                "required": false,
+                "visible": false,
+                "editable": true
+            },
+            {
+                "title": "UserPassEncoding",
+                "type": "text",
+                "name": "userPassEncoding",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "utf-8"
+            },
+            {
+                "title": "Verify SSL",
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "required": false,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "view_table",
+            "title": "View Table",
+            "description": "ipAclNewCfgTable.html: View Table",
+            "enabled": true,
+            "category": "utilities",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "parameters": [],
+            "open": false
+        },
+        {
+            "operation": "view_table_element",
+            "title": "View Table Element",
+            "description": "ipAclNewCfgTable.html: View Table Element",
+            "category": "utilities",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "indx",
+                    "type": "integer",
+                    "name": "indx",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": null,
+                    "tooltip": "The index of the IP ACL table. Note: This table is not supported in VX instance of Virtualization."
+                }
+            ],
+            "open": false
+        },
+        {
+            "operation": "edit_table_element",
+            "title": "Edit Table Element",
+            "description": "ipAclNewCfgTable.html: Edit Table Element",
+            "category": "utilities",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "Ip",
+                    "type": "text",
+                    "name": "ip",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "",
+                    "tooltip": "The IP address of the entry.\nexample)  1.1.1.1"
```

Comment:
Please provide the descriptions of all action parameters. This will help us to generate the documentation of the connector.

Reply:
there is no online documentation available along with the description.